client/src/App.tsx

```diff
@@ -103,7 +103,7 @@ const WrappedApp = new ComponentWrapper(RootNavigator)
   .wrap(ActionSheetProviderWithChildren, {})
   .wrap(AuthProvider, {})
   .wrap(Suspense, {fallback: <CustomLoadingIndicator />})
-  .wrap(ResettableRelayProvider, {createRelayEnvironment})
+  .wrap(RelayEnvironmentProvider, {environment: relayEnvironment})
```

Comment:
#### Suggestion
How about we do this?
```ts
const environment = createRelayEnvironment();
const WrappedApp = new ComponentWrapper(RootNavigator)
  // ...
  .wrap(RelayEnvironmentProvider, {environment})
```
Then we do not need to export `relayEnvironment` from `./relay`.

Reply:
@0916dhkim good news! and  the reason that export `relayEnvironment` is to use same environment instance globally. for example, `commitLocalUpdate` should get  the same`relayEnvionment` instance to invalidate records. but I'm not sure there's other way to get a environment inside `Provider`.

Reply:
@DevYuns You can use `useRelayEnvironment` hook to access environment.
https://relay.dev/docs/api-reference/use-relay-environment/

Reply:
@0916dhkim nice! I will fix it now!